lib/siftsciex/verification.ex

```diff
@@ -0,0 +1,116 @@
+defmodule Siftsciex.Verification do
+  @moduledoc """
+  The Verification module is used for using the Sift Verification Product for sending 2FA Codes to your users
+
+  For more information please read their Verification Integration Guide: https://docs.google.com/document/d/11RK_oss2Z7TjydnugYHlG5tK41GMpCyfbNbjzZAYSfM/edit
+
+  """
+
+  alias Siftsciex.Verification.{Response, Transport}
+  alias Siftsciex.Verification.Payload.{Check, Send, Resend}
+
+  @type error_map :: %{required(String.t) => String.t}
+  @type result :: {:ok, Response.t}
+                  | {:error, :redirected, String.t}
+                  | {:error, :client_error, integer}
+                  | {:error, :server_error, integer}
+                  | {:error, :transport_error, any}
```

Comment:
I don't where `:client_error`, `:server_error` and `:redirected` comes from.
`process_response` is either returning a `{:ok, Response.t}` or `{:error, :transport_error, any}`.
Did I miss something?

Reply:
Hmm I will have to take a look at that. I just copied this from the legacy code and didn't look at it that carefully. Will get back to you. Thanks.

Reply:
Thanks @rhnonose , I removed the unused return :error types.